rustler/src/export.rs

```diff
@@ -17,21 +17,14 @@
 #[macro_export]
 #[deprecated(since = "0.22.0", note = "Please use `init!` instead.")]
```

Comment:
Maybe the deprecation warning should advise to use `rustler::init!` (full path to access the function) instead to make it easier to find.